RoFILib/configuration/Generators.cpp

```diff
@@ -205,6 +220,8 @@ void next(const Configuration& config, std::vector<Configuration>& res, unsigned
     std::vector<Action> actions;
     if (bound == 1) {
         generateSimpleActions(config, actions, step);
+    } else if (bound == 2) {
+        generateBisimpleActions(config, actions, step);
```

Comment:
What's the difference between the general case and 2?

Reply:
I will change it back, it has different behaviour than general case. I will also slightly change doc of bisimpleNext to reflect that.